moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -1071,6 +1071,8 @@ bool TrajectoryExecutionManager::configure(TrajectoryExecutionContext& context,
     // empty trajectories don't need to configure anything
     return true;
   }
+
+  reloadControllerInformation();
```

Comment:
Why is this needed here?

Reply:
This's related to https://github.com/ros-controls/ros2_control/issues/464 causing `known_controllers_` to have an empty vector of joints for the newly activated controller, calling this will update the joint list